### ISSUE TITLE
Plugin stopped working?

### ISSUE THREAD

**Issue by gkiely** [keithamus/eslint-plugin-escompat#18]
I'm probably doing something dumb, but I can't seem to get it working.

Steps:

```bash
git clone git@github.com:gkiely/vite-react-ts.git
# Uncomment line 6 and 12 in App.tsx

npm i
npm run lint
# No errors shown
```

**Comment by keithamus** [comment]
I am unable to reproduce this:

```
npm run lint

> vite-react-ts@0.0.0 lint
> eslint .eslintrc.json --ext ts,tsx src


/home/keith/Code/vite-react-ts/src/App.tsx
  6:7  warning  'fn' is assigned a value but never used  @typescript-eslint/no-unused-vars

✖ 1 problem (0 errors, 1 warning)
```

Additionally you're unlikely to get any errors from escompat as you don't have a `.browserslistrc` or a `browserslist` field in your `package.json`.


I'll close this for now. Let me know if you experience any other issues with this!

**Comment by gkiely** [comment]
I added a browserlist field and updated the example.

This is what I see:
<img width="471" alt="image" src="https://user-images.githubusercontent.com/1948935/196755953-d4c18ac5-e918-431f-bb8f-2dd303276516.png">


I was expecting to see an error for using `0n`, should i not?
